packages/ui/sputnik-ui-angular/projects/sputnik/src/lib/widgets/code-widget.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input, OnInit } from '@angular/core'
+import { WidgetComponent } from './widget.component'
+
+@Component({
+	template: `
+		<div *ngIf="data">
+			<pre>
+				<code [innerHTML]="value"></code>
```

Comment:
We should scrub this for XSS. But since currently  use provides in blueprint and not controlled anywhere else is ok for now